resources/assets/tconstruct/models/block/drying_rack.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
The reason I had this file was for the sake of resource packs that wanted different models for item/drying racks, as under the current system the model has to point to hardcoded path.